docs/blog/2019-04-19-gatsby-why-we-write/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Gatsby: Why We Write
+date: 2019-04-19
+author: Sam Bhagwat
+excerpt: "Why does the Gatsby team and community write so much? Six principles that explain our logorrhea."
```

Comment:
> logorrhea

I will admit I had to look this up, and it was still a bit cryptic to me: "a tendency to extreme loquacity", in addition to referencing a communication disorder. Is there a more approachable word we can use? How about "our approach to content"?

Reply:
How about _love for writing_? Introduces a more positive tone. I do love _logorrhea_ (it's a [winning spelling bee word](https://en.wikipedia.org/wiki/72nd_Scripps_National_Spelling_Bee)!).

Reply:
That seems like a good in between spot!